esp/src/src-react/components/forms/Fields.tsx

```diff
@@ -677,18 +679,17 @@ export interface GroupMembersProps extends Omit<AsyncDropdownProps, "options"> {
 
 export const GroupMembersField: React.FunctionComponent<GroupMembersProps> = (props) => {
 
-    const [users, setUsers] = React.useState<IDropdownOption[]>();
+    const [users, setUsers] = React.useState<IDropdownOption[]>([]);
 
     React.useEffect(() => {
         const request = { groupname: props.groupname };
         WsAccess.GroupMemberEditInput({ request: request }).then(({ GroupMemberEditInputResponse }) => {
-            const _users = GroupMemberEditInputResponse?.Users?.User
-                .map(user => {
-                    return {
-                        key: user.username,
-                        text: user.username
-                    };
-                }) || [];
+            const usersArray = GroupMemberEditInputResponse?.Users?.User || [];
+            const _users = Array.from(usersArray, user => ({
+                key: user.username,
+                text: user.username
+            }));
+            _users.sort((a, b) => a.text.localeCompare(b.text));
```

Comment:
The sort() can be chained from the map()

<pre>
.map(user => {
  ...
}).sort(...)
|| [];
</pre>

Reply:
Updated, and tested.